lib/cog/queries/user.ex

```diff
@@ -9,6 +9,14 @@ defmodule Cog.Queries.User do
     preload: [:chat_handles]
   end
 
+  def for_chat_provider_user_id(chat_provider_user_id, provider) when is_binary(provider) do
```

Comment:
Just realized we're still using handles to lookup aliases so another :+1: for this suggestion.